### ISSUE TITLE
Fire Emblem : Shadow Dragon - Red Screen

### ISSUE THREAD

**Issue by Fu2chN** [DS-Homebrew/nds-bootstrap#1356]
**Your console:** New 3DS XL

**Launch Method:** SD Card

**Version used:** TWiLight Menu++: v24.1.0
nds-bootstrap: Release v0.54.1

**Game tested/used/applicable:** FIREEMBLEM11_YFEP01_00.trim.nds
Ver. SDK: 4.2  NTR-YFEP-EUR
Title ID: 0004800459464550

**Expected Behavior:** The game start correctly.

**Actual Behavior:** The game not start, red screen on boot

### Steps to reproduce
1. Juste launch the game by TWiLight Menu++ or the forwarder.

### nds-bootstrap.ini
```[NDS-BOOTSTRAP]
NDS_PATH = sd:/roms/ds/FIREEMBLEM11_YFEP01_00.trim.nds
SAV_PATH = sd:/roms/ds/saves/FIREEMBLEM11_YFEP01_00.trim.sav
BOOST_CPU = 0
DEBUG = 0
LOCK_ARM9_SCFG_EXT = 0
CACHE_FAT_TABLE = 0
BOOST_VRAM = 0
CARD_READ_DMA = 1
ASYNC_CARD_READ = 0
DSI_MODE = 1
SWI_HALT_HOOK = 1
EXTENDED_MEMORY = 0
DONOR_SDK_VER = 0
PATCH_MPU_REGION = 0
PATCH_MPU_SIZE = 0
CONSOLE_MODEL = 2
LANGUAGE = -1
LOGGING = 1
REGION = -2
RAM_DRIVE_PATH = sd:/null.img
GUI_LANGUAGE = fr
USE_ROM_REGION = 1
FORCE_SLEEP_PATCH = 0
```
### Logs
[NDSBTSRP.LOG](https://github.com/DS-Homebrew/nds-bootstrap/files/8118583/NDSBTSRP.LOG)

### Screenshots
[1645549745734](https://user-images.githubusercontent.com/13276983/155182718-f239568a-fb95-4491-ab74-0fe83540f86d.jpg)

**Comment by RocketRobz** [comment]
Your ROM is most likely corrupt, as the game boots on my side.
Try replacing your ROM, and/or turn off cheats.

**Comment by Fu2chN** [comment]
> 
mmh, i'dump my rom by gm9 it's a correct way ?

**Comment by RocketRobz** [comment]
Yes. Are cheats enabled?

**Comment by Fu2chN** [comment]
All setting are Default

**Comment by RocketRobz** [comment]
Both the USA and EUR versions boot on my side, all with default settings and no cheats.
Try re-dumping your ROM, as it might've got corrupt somehow.

**Comment by Fu2chN** [comment]
Yep, I'm look that.

**Comment by Fu2chN** [comment]
Ok my gamecard have a probleme. I tested why other fire emblem rom and it's work.
Thx for this app ^^

**Comment by RocketRobz** [comment]
You're welcome!

**Comment by Fu2chN** [comment]
I fixed the bug by updating godmode9 to 2.1.0 (I was on 1.9.3pre1).

**Comment by RocketRobz** [comment]
> I fixed the bug by updating godmode9 to 2.1.0 (I was on 1.9.3pre1).

Ah, nice. Glad to hear it's fixed!
